docs/syntax/scss.md

```diff
@@ -278,7 +278,23 @@ Modules that make use of styles defined in other modules that support silent mod
 		@include oAnotherThingFoo();
 	}
 
-Finally, in documentation, modules *must* provide information about both silent and non-silent methods, where supported, and must put the default first (i.e. if silent mode is by default **on**, the module must document the silent mode integration first).
+When a module that supports silent mode is used with silent mode turned off (i.e. `$o-{modulename}-is-silent: false`), after the module has output the CSS, it *must* also reset the value of `$o-{modulename}-is-silent` to true, (this prevents the accidental output of styles if the module is included a second time):
```

Comment:
Take for example Application A that has a dependency B, and there´s also a dependency C that's imported in both A and B.

A turns C silent mode off, and imports it, and afterwards imports B. When B imports C, as silent mode is still of because it´s a global variable, C will output its classes again, even if B is using C only via mixins